### PR TITLE
Allow Backpack in MAVLink mode to come back to Normal mode automatically

### PR DESCRIPTION
**Pull request by MUSTARDTIGERFPV**
Needs companion Backpack PR - this makes it so that going from MAVLink to Normal link mode moves the backpack too.

### CODE REVIEW

File: src/lib/Backpack/devBackpack.cpp [hunk 1]
```diff
@@ -362,6 +365,22 @@ static int event()
         // EN should be HIGH to be active
         digitalWrite(GPIO_PIN_BACKPACK_EN, config.GetBackpackDisable() ? LOW : HIGH);
     }
+#endif
+#if !defined(PLATFORM_STM32)
+  // Update the backpack operating mode when the link mode changes
+    uint8_t newMode = config.GetLinkMode();
+    if (lastLinkMode != newMode)
+    {
+        if (lastLinkMode == TX_NORMAL_MODE) {
```
Comment (JyeSmith) [inline src/lib/Backpack/devBackpack.cpp:374]: Bracket newline for consistency.

File: src/lib/Backpack/devBackpack.cpp [hunk 2]
```diff
@@ -362,6 +365,22 @@ static int event()
...
+            // Backpack will autodetect MAVLink for now - no reason to send an MSP message
+        } else if (lastLinkMode == TX_MAVLINK_MODE) {
```
Comment (JyeSmith) [inline src/lib/Backpack/devBackpack.cpp:376]: Bracket

File: src/lib/LUA/tx_devLUA.cpp [hunk 3]
```diff
@@ -707,6 +707,7 @@ static void registerLuaParameters()
       if (isDisconnected)
       {
         config.SetLinkMode(arg);
+devicesTriggerEvent(); // trigger event to let the backpack know the link mode has changed (see devBackpack)
```
Comment (pkendall64) [inline src/lib/LUA/tx_devLUA.cpp:710]: Should not need this as the devices are notified once the config is committed in the main loop.

Comment (MUSTARDTIGERFPV) [reply src/lib/LUA/tx_devLUA.cpp]: So you're right - done. :)

**Review by pkendall64 — APPROVED** [review]

**Review by JyeSmith — APPROVED** [review]

**Pull request merged**